hackathon_site/hardware/views.py

```diff
@@ -4,7 +4,7 @@
 
 
 class HardwareListView(mixins.ListModelMixin, generics.GenericAPIView):
-    queryset = Hardware.objects.all()
+    queryset = Hardware.objects.get_queryset()
```

Comment:
This can stay as `Hardware.objects.all()` to stick with convention. If you check out the [source code](https://github.com/django/django/blob/main/django/db/models/manager.py#L149) for the `all` method, it just calls `get_queryset()`, which you extended in your custom manager.